Removed a few rules of my prior Pull request + many new rules.

Evonos opened the pull request:
Multiple new rules + removed a rule which isnt necessary anymore.

not 100% sure i remember all changes ( pull 16 was a few weeks old till i closed it to make pull 17 )

Optimization Meats / leathers Mods Load bottom ( as they are Problematic mods , not only for converting different things into their own meats and leathers , but also other recipes need to adjust )

Revia race Biotech version before the Revia race Facial animation ( it only knows about the HAR version )

and more.

Most of the changes should have comments if not all , i can make a new fork and get the changes from there should be easier to see the differences.

LionelColaso commented:
Please add what mod rules you are adding or removing in the pull request details since that becomes easy to check what has to be expected in the changes for everyone
Thanks

Evonos commented:
> Please add what mod rules you are adding or removing in the pull request details since that becomes easy to check what has to be expected in the changes for everyone Thanks

added some info in the first , but yeah the initial pull reuqest ( 16 ) was a bit older and i fixed a few mods loading order between got even more waiting after , ( running a modlist of 370+ atm which needed some fixing with rimsorts initial sorting )
